frame/contracts/src/benchmarking/mod.rs

```diff
@@ -1971,6 +1970,47 @@ benchmarks! {
 		let origin = RawOrigin::Signed(instance.caller.clone());
 	}: call(origin, instance.addr, 0u32.into(), Weight::MAX, None, vec![])
 
+	#[skip_meta]
```

Comment:
We don't need it here. We write many times to the same map which does not spam.

Reply:
Updated